components/list/ListItem.tsx

```diff
@@ -1,15 +1,7 @@
 import Image from '../Image';
-import {
-    ListItem as MUIListItem,
-    ListItemAvatar,
-    Avatar,
-    ListItemText,
-    Grid,
-    Typography,
-    makeStyles,
-    createStyles,
-    Theme,
-} from '@material-ui/core';
+import { ListItem as MUIListItem, ListItemAvatar, Avatar, ListItemText, Grid, Typography, Theme } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
```

Comment:
Could we combine these from `@mui/styles`?
```suggestion
import { makeStyles, createStyles  } from '@mui/styles;
```